### ISSUE TITLE
Example in readme shows time as a number but seems to be required to be string

### ISSUE THREAD

**Issue by microbit-matt-hillsdon** [eddiewebb/circleci-queue#18]
The example in the readme shows the time parameter as a YAML number, but it's actually [required to be a string](https://github.com/eddiewebb/circleci-queue/blob/fa5bb32714ade972c6e62880aae75ea1dd9b7fc0/src/commands/until_front_of_line.yml#L12). I got an error with an unquoted number in my CI config:

```
# Error calling command: 'queue/until_front_of_line'
# Type error for argument time: expected type: string, actual value: \"60\" (type integer)
```

It worked fine with `time: "60"`

Happy to send a PR if it's the docs that need updating.

**Comment by eddiewebb** [comment]
Thanks for catching this!

Ultimately the bash script won't care, so I am open to switching parameter to be `integer` type, or update the docs to use a string.  The former feels cleaner now that its a valid [parameter type](https://circleci.com/docs/2.0/reusing-config/#parameter-types).

**Comment by eddiewebb** [comment]
Fixed in #20
